MusicTaste/MusicTaste/View Controllers/ProfileViewController.m

```diff
@@ -3,8 +3,6 @@
 //  MusicTaste
```

Comment:
Did you want to remove these as well? Either way; the header comments at the top are ok or can be removed, but would be good to be consistent.